cockroachcloud/use-managed-service-backups.md

```diff
@@ -25,7 +23,7 @@ Cockroach Labs runs [full cluster backups](../{{site.current_cloud_version}}/tak
 {{site.data.alerts.callout_info}}
```

Comment:
I can take a look before you include in a commit, if that's what you prefer! Feel free to send me the shots directly

Reply:
Your dedicated screenshot made me double check the UI, it looks like **Status** doesn't exist in that UI any longer, so that can be deleted. Might be good to include a screenshot showing incrementals and full backups for dedicated: 

<img width="845" alt="image" src="https://user-images.githubusercontent.com/44557882/211918076-98d741c7-3bdd-4828-974a-9fddacd61186.png">

Feel free to use that one ^